projects/core/src/anonymous-consents/store/effects/anonymous-consents.effect.spec.ts

```diff
@@ -98,11 +98,14 @@ const mockUserToken = {
   access_token: 'yyy',
 } as UserToken;
 
-const mockAnonymousConsentsConfig: AnonymousConsentsConfig = {
+const mockAnonymousConsentsConfig = {
```

Comment:
we lose type safety from merging them, 

is it better to separate them and have AnonymousConsentsConfig typesafety and and its own object mockFeaturesConfig with FeaturesConfig as type?